css/default.css

```diff
@@ -61,7 +59,6 @@ a:hover {
 
 #content {
   padding-top: 1em;
-  border-top: 1px dotted #808000;
```

Comment:
No, it just felt like an extra separator. The fact that you have your name, large and at the top along with the navigation creates a visual block which separates it from the rest of the page. This dotted line separates it again so I removed it to open it up a little.

Up to you, honestly.